python/benchmarks/oxford_pet.py

```diff
@@ -6,17 +6,29 @@
 import duckdb
 import numpy as np
 import pandas as pd
-
-import lance
 import pyarrow as pa
 import pyarrow.compute as pc
 import pyarrow.dataset
 from bench_utils import BenchmarkSuite, download_uris
 from parse_pet import OxfordPetConverter
 
+import lance
+
 oxford_pet_benchmarks = BenchmarkSuite("oxford_pet")
 
 
+@oxford_pet_benchmarks.benchmark("total_count", key=['fmt', 'flavor'])
+def total_count(base_uri: str, fmt: str, flavor: Optional[str]):
+    if fmt == "raw":
+        return get_pets_class_distribution(base_uri)
```

Comment:
This doesn't count number of rows ?

Reply:
Oh, i copied this piece of code from class distribution, and have not add raw performance number.

Will add it later.